src/main/resources/jenkins/plugins/openstack/compute/OpenStackDSL/help.jelly

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core">
+    <p>
+        The <code>openstack</code> variable offers convenient access to Openstack-related functions from a Pipeline script.
```

Comment:
In fact, this help.jelly is also invalid (to be deleted) as it refers to the previous groovy DSL implementation